src/components/App.js

```diff
@@ -36,9 +44,13 @@ class App extends React.Component {
           <div className="slate">
             <h2> Score </h2>
             <ScoreBoard></ScoreBoard>
+            <h2> Room Info </h2>
+            <p> Room ID : {room}</p>
             <h2> Letters </h2>
             <Slate></Slate>
+            <button onClick = { this.onSubmit }> See Game Rules </button>
```

Comment:
The button is not visible in the Rules component, one does not have any means to return.
And modify the button's text based on the state.

Reply:
Yeah I didn't put the button on the rules side because I am confused if i render the app component it will also render the board again all the tiles are misplaced. I think 

I will look into that tommorow.